client/verta/tests/test_entities.py

```diff
@@ -91,6 +91,14 @@ def test_else_http(self):
 
     @pytest.mark.skipif('VERTA_EMAIL' not in os.environ or 'VERTA_DEV_KEY' not in os.environ, reason="insufficient Verta credentials")
     def test_config_file(self):
+        self.test_config_file_with_type(connect = False)
+
+    @pytest.mark.skipif('VERTA_EMAIL' not in os.environ or 'VERTA_DEV_KEY' not in os.environ, reason="insufficient Verta credentials")
+    def test_config_file_connect(self):
+        self.test_config_file_with_type(connect = True)
+        
+    @pytest.mark.skip("util function")
+    def test_config_file_with_type(self, connect):
```

Comment:
Instead of `mark.skip`, I suggest simply renaming this function to something that does _not_ start with `test_` (for example: `def config_file_test(self, connect)`)

Pytest will then ignore it, because Pytest only runs functions that start with `test_`!

Reply:
fixed